main.tf

```diff
@@ -126,17 +127,19 @@ data "aws_iam_policy_document" "codebuild" {
 }
 
 module "build" {
-  source    = "git::https://github.com/cloudposse/tf_codebuild.git?ref=tags/0.1.0"
-  namespace = "${var.namespace}"
-  name      = "${var.name}-build"
-  stage     = "${var.stage}"
-
+  source        = "git::https://github.com/cloudposse/tf_codebuild.git?ref=tags/0.3.0"
+  namespace     = "${var.namespace}"
+  name          = "${var.name}"
+  stage         = "${var.stage}"
   image         = "${var.build_image}"
   instance_size = "${var.build_instance_size}"
+  delimiter     = "${var.delimiter}"
+  attributes    = "${lower(join(var.delimiter, compact(concat(var.attributes, list("build")))))}"
```

Comment:
Attributes is an array, not a string. join returns a string. Should simply concat arrays and do not worry about lowering, joining.